parachain/src/command.rs

```diff
@@ -51,7 +51,8 @@ impl IdentifyChain for dyn sc_service::ChainSpec {
         self.id().starts_with("interlay")
     }
     fn is_kintsugi(&self) -> bool {
```

Comment:
Where is this used? Just want to make sure we're not accidentally changing any behavior. In https://github.com/interlay/interbtc/pull/534 I intentionally left this file unchanged